manifests/foreman.pp

```diff
@@ -18,6 +18,7 @@
 ) inherits certs {
 
   $client_cert_name = "${hostname}-foreman-client"
+  $client_dn = "CN=${hostname}, OU=PUPPET, O=FOREMAN, ST=${state}, C=${country}"
```

Comment:
city is also a parameter. Is that `undef` by default so it doesn't show up? Do we have users who do pass in `$city`?

How about adding `$org` and `$org_unit` as class parameters and reusing them here?

Reply:
`$city` is defined, it just does not appear in the DN. I've never thought to go look why if its a parameter though. That should not hold this up but I could go investigate it separately. I've added the parameters for org and org_unit.